src/components/accession2/edit/CalculatorModal.tsx

```diff
@@ -150,6 +150,7 @@ export default function CalculatorModal(props: CalculatorModalProps): JSX.Elemen
               onChange={(value) => onChange('subsetCount', value)}
               type='text'
               value={record.subsetCount}
+              disableDecimals={true}
```

Comment:
we could use type=number and set the step size to enable/disable decimals?

Reply:
I tested stepSize, but allows inputing the ',' and '.' and that is what we don't want. The step just work with the arrows